### DISCUSSION TITLE
Unhealth on Indexer service on Polkadot project

### DISCUSSION THREAD

**Discussion by roongpok** [subquery/subql#1067] [Frontier TestNet - Bug Reporting]
![Screen_Shot_2565-05-13_at_18 20 49](https://user-images.githubusercontent.com/82475020/170239977-556b2085-7838-49b1-af99-b520e152566a.png)

I need to do docker down and up to fix the issue.

**Comment by mzxyz** [comment]
You can also try to restart the indexing the project, it will restart the node service
